snmp/datadog_checks/snmp/data/default_profiles/ibm-lenovo-server.yaml

```diff
@@ -0,0 +1,156 @@
+extends:
+  - _base.yaml
```

Comment:
no need for this since `ibm.yaml` already extends `_base.yaml`
```suggestion
```

Reply:
Good catch!

Reply:
Changed prefix to `ibm.imm` as suggested by @AlexandreYang